ansible/playbook/openstack_vm_create.yml

```diff
@@ -0,0 +1,70 @@
+---
+# Variables:
+#  - "hostname": "n311-test"
+#  - "openstack": {
+#      "os_username": "'$(pass show openstack/host/console_user)'"
+#      "os_password": "'$(pass show openstack/host/console_pw)'"
+#      "os_domain":  "'$(pass show openstack/host/console_domain)'"
+#      "os_auth_url": "https://rhos-d.infra.prod.upshift.rdu2.redhat.com:13000/v3/"
+#      "vm": {
+#        "network": "provider_net_shared"
+#        "security_group": "spring-boot"
+#        "flavor": "ci.m5.large"
+#        "volumes" : ["test-volume"]
+#      }
+#    }}
+- name: "Openstack VM create"
+  hosts: localhost
+  gather_facts: no
+
+  roles:
+    - role: "openstack/vm"
+      vars: 
+        state: "present"
+
+  post_tasks:
+    - name: Refresh the inventory so the newly added host is available
+      meta: refresh_inventory
+
+- name:  "Wait for the VM to boot and we can ssh"
+  hosts: "{{ vm_name }}"
+  gather_facts: no
+
+  tasks:
+    - name: "Show 'Wait for connection to host' output"
+      debug:
+        msg: 
+          - "ip : {{ query('passwordstore', '/openstack/' + vm_name + '/ansible_ssh_host')[0] }}"
```

Comment:
Is there a control/check verifying if password store is installed on the users machine ?

Reply:
I have implemented a new playbook to validate our passwordstore when creating openstack VMs. Will later apply to the hetzner VMs when both PR are merged.